binding.cc

```diff
@@ -506,6 +520,7 @@ struct Iterator {
     if (gte_ != NULL) {
       delete gte_;
     }
+    delete options_;
```

Comment:
Without this, `node --expose-gc test/leak-tester-iterators` shows endlessly growing memory, also on master.